lib/aruba/processes/debug_process.rb

```diff
@@ -0,0 +1,59 @@
+require 'aruba/processes/spawn_process'
+
+module Aruba
+  module Processes
+    # Create process
+    #
+    # @params [String] cmd
+    #   Command string
+    #
+    # @params [Integer] exit_timeout
+    #   The timeout until we expect the command to be finished
+    #
+    # @params [Integer] io_wait
+    #   The timeout until we expect the io to be finished
+    #
+    # @params [String] working_directory
+    #   The directory where the command will be executed
+    class DebugProcess < BasicProcess
```

Comment:
This is an interesting one. Flagging a Test with `@debug` activates this launcher. This will make things like `binding.pry` in your cli application possible for a single test to debug a bug occuring during test under cucumber.

```cucumber
@debug
Feature: Blub
  Scenario: Hello World
    Given I run `cucumber`
```